com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -60,29 +108,21 @@ var FileLoader = {
         request.send();
     },
     // Do HTTP GET request
-    // onprogress(loaded, total)
+    // onprogress(loadedDelta)
     // onerror(error)
     // onload(response)
-    load: function(url, responseType, estimatedSize, onprogress, onerror, onload) {
+    // onretry(loadedSize, currentAttempt)
+    load: function(url, responseType, onprogress, onerror, onload, onretry) {
         var request = FileLoader.request(url, "GET", responseType);
-        request.onprogress = function(xhr, e) {
-            if (e.lengthComputable) {
-                onprogress(e.loaded, e.total);
-                return;
-            }
-            var contentLength = xhr.getResponseHeader('content-length');
-            var size = contentLength != undefined ? contentLength : estimatedSize;
-            if (size) {
-                onprogress(e.loaded, size);
-            } else {
-                onprogress(e.loaded, e.loaded);
-            }
+        request.onprogress = function(xhr, e, ls) {
+            var delta = e.loaded - ls;
+            onprogress(delta);
```

Comment:
Simplify `onprogress` because we don't need to know total size from response headers (see https://github.com/defold/defold/issues/7039)